pynemo/profiler.py

```diff
@@ -388,31 +388,38 @@ def write_tidal_data(setup_var, dst_coord_var, grid, tide_cons, cons):
             
             nemo_bdy_tide_ncgen.CreateBDYTideNetcdfFile(fout_tide, 
                             val['nx'], 
-                            dst_coord_var.lonlat['t']['lon'].shape[1],
-                            dst_coord_var.lonlat['t']['lon'].shape[0], 
+                            dst_coord_var.lonlat[key]['lon'].shape[1],  # "key" is the grd value
```

Comment:
I'm pretty sure this is correct: each file should have their own lat/lon, not just as defined on t-pts.

Reply:
memory is a little hazy on this... but that makes sense

Reply:
I guess the original reason for this is that NEMO traditionally uses nav_lon and nav_lat in U/V/T output files and that nav_lon and nav_lat are only ever on the T-Grid. So there may be confusion. True coordinates on the various grids are referenced as: glamt, glamu, glamv, gphit .... etc